source/_includes/asides/component_navigation.html

```diff
@@ -18,39 +18,22 @@
     </div>
   {%- endif -%}
 
-  {%- if page.ha_iot_class -%}
-    <div class='section'>
-      IoT class<sup><a href='/blog/2016/02/12/classifying-the-internet-of-things/#classifiers'><i class="icon-info-sign"></i></a></sup>: {{ page.ha_iot_class }}
-    </div>
-  {%- endif -%}
-
-  {%- if page.ha_quality_scale -%}
-    <div class='section'>
-      Quality Scale: <a href='/docs/quality_scale/'>
-        {%- if page.ha_quality_scale == 'platinum' -%}
-          🏆 (platinum)
-        {%- elsif page.ha_quality_scale == 'gold' -%}
-          🥇 (gold)
-        {%- elsif page.ha_quality_scale == 'silver' -%}
-          🥈 (silver)
-        {%- elsif page.ha_quality_scale == 'internal' -%}
-          🏠 (internal)
-        {%- endif -%}
-      </a>
-    </div>
-  {%- endif -%}
-
-  {%- if page.ha_release -%}
-    <div class='section'>
-      Introduced in release: {{ page.ha_release }}
-    </div>
-  {%- endif -%}
+  <div class='section'>
+    The {{ page.name | default: page.title }} integration was added with the {{ page.ha_release | default: "0.7" }} release.
```

Comment:
```suggestion
    The {{ page.name | default: page.title }} integration was added with the {{ page.ha_release | default: "unknown" }} release.
```
"unknown"  maybe, we will get feedback and can adjust if it's missing somewhere?